core/src/main/java/cz/xtf/core/image/Image.java

```diff
@@ -70,7 +71,10 @@ public static Image from(String imageUrl) {
                 repoTag = slashTokens[2];
                 break;
             default:
-                throw new IllegalArgumentException("image '" + imageUrl + "' should have one or two '/' characters");
+                registry = slashTokens[0];
+                user = slashTokens[1];
```

Comment:
This is different to what dosu says :) My understanding is aligned with dosu implementation.

IIUC dosu implementation https://github.com/xtf-cz/xtf/issues/572#issuecomment-2296478769 will bring this result
```
registry = mcr.microsoft.com
user = mssql/rhel
repoTag = server:2022-CU13-rhel-9.1
```

whereas your implementation will bring 

```
registry = mcr.microsoft.com
user = mssql
repoTag = rhel/server:2022-CU13-rhel-9.1
```

Reply:
to me the latter makes more sense, though we are only bending image url (which can be almost anything) to what the class expects

Reply:
Yes, you was faster then me wanted to write `user = mssql ` makes more sense to me now while thinking about it.